packages/moonstone/VirtualList/VirtualListBase.js

```diff
@@ -695,7 +651,12 @@ const VirtualListBaseFactory = (type) => {
 		focusOnItem = (index) => {
 			const item = this.uiRefCurrent.containerRef.current.querySelector(`[data-index='${index}'].spottable`);
 
-			this.pause.resume();
+			if (this.isWrappedBy5way) {
+				SpotlightAccelerator.reset();
+				this.isWrappedBy5way = false;
+				this.pause.resume();
```

Comment:
`this.pause.resume()`  should go out of `if` statement. This cause spot lost issue on hover

Reply:
Good catch - I've updated this so that spotlight resumes properly.